cli/src/lib/constants.ts

```diff
@@ -29,3 +29,5 @@ export const ADDRESS_ZERO = `0x${"0".repeat(40)}`;
 // export const ZERO_U256 = `0x${"0".repeat(64)}`;
 export const OPEN_ZEP_PROXY_IMPL_SLOT =
   "0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc";
+
+export const UPGRADE_FN_SELECTOR: Hex = "0x08284e57";
```

Comment:
Would it not be better to save this as `getFunctionSelector("upgradeTo(address)")` somewhere?

Reply:
Let's do this change.